### ISSUE TITLE
1.0.0 changed something...

### ISSUE THREAD

**Issue by jssuttles** [klei/grunt-injector#45]
What changed between 0.6.1 and 1.0.0?

**v0.6.1 OUTPUT**
Running "injector:bower" (injector) task
Missing option `template`, using `dest` as template instead
Injecting js files (1 files)
 Nothing changed

**v1.0.0 OUTPUT**
Running "injector:bower" (injector) task
Missing option `template`, using `dest` as template instead
Warning: Unable to read "undefined" file (Error code: ENOENT). Use --force to continue.

```
injector: {
      bower: {
        options: {
          template: null,
          starttag: '<!-- injector:bower:{{ext}} -->',
          endtag: '<!-- endinjector -->',
          ignorePath: 'public'
        },
        files: {
          './views/footer.ejs': ['<%= project.dest.folder %>/bundle.min.js']
        }
      }, ... 
}

project: {
      client: 'public',
      dest: {
        folder: '<%= project.client %>/dist'
      }
}
```

**Comment by luketc1** [comment]
👍 confirming that 1.0.0 had a breaking change. Tried to use this for the first time today and got the same error `Warning: Unable to read "undefined" file (Error code: ENOENT). Use --force to continue.`

**Comment by ptitgraig** [comment]
This comes from a non-handle case while trying to figure out line ending. Thanks for reporting. I'll fix it asap.

**Comment by chillyistkult** [comment]
Is the fix already integrated in v1.0.0?
